meta-mel/conf/distro/mel.conf

```diff
@@ -184,9 +184,8 @@ PACKAGECONFIG:append:pn-pulseaudio = " autospawn-for-root"
 PACKAGECONFIG_REMOVE:pn-plymouth = "initrd"
 ## }}}1
 ## Inherits {{{1
-# We want information about package and image contents
+# We want information and history about build output
 INHERIT += "buildhistory"
-BUILDHISTORY_COMMIT ?= "1"
```

Comment:
Again, should we rely on the upstream default or keep it as a distro decision?

Reply:
In this case I don't think it's necessary, since we don't explicitly require this particular feature for our product.